packages/widget/src/ui/HistoryDialog/HistoryList.tsx

```diff
@@ -238,11 +235,9 @@ export const Item = forwardRef<HTMLDivElement, ItemProps>(function Item(
             </DescriptionList.Row>
           ))}
           <DescriptionList.Row>
-            <DescriptionList.Dt>Completion Time</DescriptionList.Dt>
+            <DescriptionList.Dt>Estimation Completion Time</DescriptionList.Dt>
```

Comment:
is this the right text? should it be "Estimated Completion time"?

Reply:
yeah because we don't know the real completion time

Reply:
@codingki I'm asking if it should be "Estimated" instead of "Estimation", not a big deal but Estimation Completion Time sounds weird to me

Reply:
hmm you right, I'll fix the typo before release